tests/integ/sagemaker/experiments/conftest.py

```diff
@@ -0,0 +1,177 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit empty __init__.py either may not be needed at all or add the generic copyright header(which is obsolete?)

Reply:
To make the integ/sagemaker/experiments as a module, the __init__.py file is required. Added the copyright header